api/gradlew.bat

```diff
@@ -1,3 +1,19 @@
+@rem

```

Comment:
If we don't support windows or cygwin, do we still need this file and the cygwin section above?